.eslintrc

```diff
@@ -25,221 +32,152 @@
     }
   },
   "rules": {
-    "react/jsx-no-undef": 0,
+    "react-hooks/exhaustive-deps": "error",
+    "react/jsx-boolean-value": ["error", "never", { "always": [] }],
+    "react/jsx-closing-bracket-location": ["error", "line-aligned"],
+    "react/jsx-closing-tag-location": "error",
+    "react/jsx-curly-spacing": ["error", "never", { "allowMultiline": true }],
+    "react/no-arrow-function-lifecycle": "error",
+    "react/no-invalid-html-attribute": "error",
+    "react/jsx-no-useless-fragment": "error",
+    "react/jsx-no-constructed-context-values": "error",
+    "react/jsx-fragments": ["error", "syntax"],
+    "react/jsx-no-duplicate-props": ["error", { "ignoreCase": true }],
+    "react/jsx-pascal-case": [
+      "error",
+      {
+        "allowAllCaps": true,
+        "ignore": []
+      }
+    ],
+    "react/no-danger": "warn",
+    "react/no-did-update-set-state": "error",
+    "react/no-will-update-set-state": "error",
+    "react/self-closing-comp": "error",
+    "react/jsx-no-undef": ["error", { "allowGlobals": true }],
```

Comment:
I would also consider `react/jsx-no-leaked-render` and ``react/no-object-type-as-default-prop` as they can early catch some pretty nasty bugs

Reply:
These require an update to `eslint-plugin-react` and introduce ~100 new errors. I'll create an issue to enable and fix these rules in the interest of keeping the scope of this PR down